client/components/link-card/index.tsx

```diff
@@ -0,0 +1,71 @@
+import styled from '@emotion/styled';
+import { ReactChild } from 'react';
+
+import './style.scss';
+
+interface LinkCardContainerProps {
+	background: string;
+}
+
+interface LinkCardProps {
+	label: ReactChild;
+	title: ReactChild;
+	cta: ReactChild;
+	background: string;
+	url: string;
+}
+
+const LinkCardContainer = styled.div< LinkCardContainerProps >`
+	border-radius: 5px;
+	padding: 24px;
+	background: var( --${ ( props ) => props.background } );
+`;
+
+const LinkCardLabel = styled.div`
+	margin-bottom: 8px;
+	font-size: var( --scss-font-body-extra-small );
+	color: var( --color-text-inverted );
+	line-height: 1.25rem;
+	text-transform: capitalize;
```

Comment:
I wasn't sure about this one but going by [this SO answer](https://stackoverflow.com/a/21052849), it looks like we probably want to just let translators handle capitalization.

```suggestion
```

Reply:
Good catch 👍🏼

Reply:
✅